src/tests/ftest/util/dmg_utils.py

```diff
@@ -547,9 +547,12 @@ def storage_scan(self):
         self.sub_command_class.set_sub_command("scan")
         return self._get_result()
 
-    def storage_format(self):
+    def storage_format(self, reformat=False):
         """Get the result of the dmg storage format command.
 
+        Args:
+            reformat (bool): always reformat storage, could be destructive.
```

Comment:
specifically this will recreate control-plane related metadata (superblock) and reformat if the media is available and formattable

Reply:
I'll make this change.